core/arch/arm/plat-stm32mp1/conf.mk

```diff
@@ -254,6 +254,15 @@ CFG_WITH_NSEC_UARTS ?= y
 # UART instance used for early console (0 disables early console)
 CFG_STM32_EARLY_CONSOLE_UART ?= 4
 
+# Disable the HUK by default as it requires CFG_STM32MP1_HUK_BSEC_BASE
+# which is product specific
+CFG_STM32MP1_HUK_BSEC ?= n
```

Comment:
I would prefer to call the switch with CFG_STM32MP15_*** as, STM32MP13 as a real HUK that will be managed in another way.